README.md

```diff
@@ -636,6 +661,8 @@ the intent:// scheme. In this intent:// scheme the RP can set the package
 name of the RP app. Since we started the flow inside the RP app the user
 will be redirected to the legit app.
 
+JJH: I think the 'since' makes the assumption in-app browser tabs are supported by the user's browser? If not I don't understand it. Or is the assumption that the flow started in the genuine RP app? I'm not sure we can guarantee that. We can guarantee the user doesn't end up back in a browser/malicious app with a different package id, but it's upto the RP to verify if the RP app in use is genuine.
```

Comment:
I wanted to say that if the flow starts in the legit RP app we would end up in the legit RP app in the app2web case at the end. But you are right, I assume here that the RP can somehow verify that no one tampered with their app.

Reply:
Thanks - I've pushed a change that tries to express that.